CI fails on RO validation

Since 13 April, the CI pipeline fails in the process-artifact job. It seems the problem stems from an upgrade in rocrate package which changs the rocrate conformity version from 1.1 to 1.2.

```
AssertionError: RO-Crate is invalid!
Detected issue of severity REQUIRED with check "ro-crate-1.1_5.3": The RO-Crate metadata file descriptor MUST have a `conformsTo` property with the RO-Crate specification version
```

--- a/environment_benchmarks.yml
+++ b/environment_benchmarks.yml
@@ -11,4 +11,4 @@ dependencies:
   - conda
   - pip
   - pip:
-      - "git+https://github.com/izus-fokus/snakemake-report-plugin-metadata4ing@v1.2.5#egg=snakemake-report-plugin-metadata4ing"
+      - "git+https://github.com/izus-fokus/snakemake-report-plugin-metadata4ing@v1.2.6#egg=snakemake-report-plugin-metadata4ing"

